Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al

```diff
@@ -299,21 +299,22 @@ codeunit 20352 "Connectivity App Definitions"
         AppApprovedFor: Text;
     begin
         /***************************************************
-            Add app 'Bankfeed - Bank Statement Import & Reconciliation' to LT, DK, HU, GB
+            Add app 'Bankfeed - Bank Statement Import & Reconciliation' to LT, DK, HU, GB, ES
         ***************************************************/
 
-        AppId := '8051ee19-8246-489f-9f2c-4d89e1710f7e';
+        AppId := '74689c09-2ed3-4e69-a2e8-c9310a271b9a';
         AppName := 'Bankfeed - Bank Statement Import & Reconciliation';
         AppPublisher := 'Softera Baltic';
         AppDescription := 'Bank statement import & reconciliation.';
-        AppProviderSupportURL := 'https://www.softera.lt/en/supported-banks-softera-bankfeed/';
-        AppSourceUrl := 'https://appsource.microsoft.com/en-us/product/dynamics-365-business-central/PUBID.softera_baltic%7CAID.softeradokubank%7CPAPPID.8051ee19-8246-489f-9f2c-4d89e1710f7e';
-        AppApprovedFor := 'LT,DK,HU,GB';
-        AppWorksOn := 'W1,DK,GB';
+        AppProviderSupportURL := 'https://bankfeed.com/banks/';
+        AppSourceUrl := 'https://appsource.microsoft.com/en-us/product/dynamics-365-business-central/PUBID.softera_baltic%7CAID.softeradokubank%7CPAPPID.74689c09-2ed3-4e69-a2e8-c9310a271b9a';
+        AppApprovedFor := 'LT,DK,HU,GB,ES';
```

Comment:
We don't yet see enough customer uptake (3 tenants) in ES (Spain). 
We do see enough uptake in EE (Estonia). Could it be that you meant to add Estonia and not Spain?

Do you want to add Latvia (LV) as well, or are you not ready to do so yet? (It has enough uptake).

Reply:
There must be some differences in customer uptake counting. For now, we removed Spain from the list. And will make another pull request after we figure out how to properly count the uptake.

But changes to AppId and URLs are important, so please confirm those. Thank you.

Reply:
Thank you for the changes. Looks good now from a AppSource URL and country perspective.